tests/science/global_param.classic.5.0_snotel.txt

```diff
@@ -0,0 +1,90 @@
+NLAYER                3
+NODES                 10
+MODEL_STEPS_PER_DAY   24
+SNOW_STEPS_PER_DAY    24
+RUNOFF_STEPS_PER_DAY  24
+STARTYEAR             2007
+STARTMONTH            11
+STARTDAY              01
+STARTSEC              00
+ENDYEAR               2008
+ENDMONTH              07
+ENDDAY                31
+
+FULL_ENERGY           TRUE
+FROZEN_SOIL           FALSE
+
+QUICK_FLUX            FALSE
+
+FORCING1              $test_data_dir/datasets/snotel/forcings_vp/snotel_VIC.4.1.2_forcings_
+FORCE_FORMAT          ASCII
+FORCE_TYPE            SWDOWN
+FORCE_TYPE            LWDOWN
+FORCE_TYPE            AIR_TEMP
+FORCE_TYPE            PRESSURE
+FORCE_TYPE            PREC
+FORCE_TYPE            VP
+FORCE_TYPE            WIND
+
+FORCE_STEPS_PER_DAY   24
+FORCEYEAR             2007
+FORCEMONTH            11
+FORCEDAY              01
+GRID_DECIMAL          4
+WIND_H                17.5
+
+SOIL                  $test_data_dir/datasets/snotel/parameters/Snotel_VIC.4.1.2_soil.txt
+BASEFLOW              ARNO
+
+JULY_TAVG_SUPPLIED    FALSE
+ORGANIC_FRACT         FALSE
+
+VEGLIB                $test_data_dir/datasets/snotel/parameters/Snotel_VIC.4.1.2_veglib.txt
+VEGPARAM              $test_data_dir/datasets/snotel/parameters/Snotel_VIC.4.1.2_vegparam.txt
+ROOT_ZONES            3
+
+VEGPARAM_LAI          TRUE
+LAI_SRC               FROM_VEGPARAM
+
+SNOW_BAND             1
+
+RESULT_DIR            $result_dir
+
+OUTPUT_STEPS_PER_DAY  1
+SKIPYEAR              0
+COMPRESS              FALSE
+OUT_FORMAT            ASCII
+ALMA_OUTPUT           FALSE
+MOISTFRACT            FALSE
+PRT_HEADER            TRUE
+PRT_SNOW_BAND         FALSE
+
+N_OUTFILES  1
+
+OUTFILE         outfile 24
```

Comment:
update the output options for VIC 5 -- see the documentation for current options.